src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -165,6 +169,16 @@ else if (DBConstants.DDP_ABSTRACTION_ALIAS.equals(source)) {
         }
     }
 
+    List<String> getGovernedUserIdsFromElasticList(List<ElasticSearchParticipantDto> proxyData) {
+        List<String> participantGovernedUserIds = new ArrayList<>();
+
+        proxyData.stream()
+                .filter(esGovernedUserData -> esGovernedUserData.getGovernedUsers().orElse(Collections.emptyList()).size() > 0)
```

Comment:
if we have `getGovernedUsers()` to return empty list instead of wrapping it in `Optional` it should be much more convenient, since we won't have to do extra work like `orElse`, `Optional` is powerful if field type is not `Collection`.